apollo-server/docker-compose.apollo.yml

```
@@ -0,0 +1,8 @@
+services:
+  apollo:
+    build:
+      context: .
+      dockerfile: Apollo.Dockerfile
+    ports:
+      - "4000:4000"
+    restart: unless-stopped
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)